themes-default/slim/views/partials/showheader.mako

```diff
@@ -121,14 +121,12 @@
                  </div>
                  <div id="tags" class="pull-left col-lg-9 col-md-9 col-sm-12 col-xs-12">
                      <ul class="tags">
-                         % if show.imdb_info.get('genres'):
+                         %if show.genre:
+                            <app-link v-for="genre in dedupeGenres(show.genres)" :href="'https://trakt.tv/shows/popular/?genres=' + genre.toLowerCase().replace(' ', '-')" :title="'View other popular ' + genre + ' shows on trakt.tv.'"><li>{{ genre }}</li></app-link>
```

Comment:
Unfortunately `show.genres` (from the API) contains both indexer genres and imdb genres (no duplicates).
Unlike `show.genre` (Directly from Python) which only contained the indexer genre...
So I don't know how well this is going to work..

Reply:
I haven't need any that don't work with the trakt link and the `dedupeGenres` function ensures it removes ones that are the same but have a hyphen instead of space.

Reply:
I said "(no duplicates)", meaning the backend already dedupes the list.
If it works, that's okay. It's not really a priority.